### PR TITLE
Add pip install options

### PR DESCRIPTION
**Pull request by fstagni**

### CODE REVIEW

**Review by atsareg — APPROVED** [review]

**Comment by fstagni** [comment]
I added a test and corrected a bug. Please re-review.

File: tests/CI/pilot_ci.sh [hunk 1]
```diff
@@ -101,6 +101,10 @@ PilotInstall(){
   if [[ "${modules}" ]]; then
     pilotOptions+=" --modules=""${modules}"
   fi
+  # shellcheck disable=SC2154
+  if [[ "${pip_install_options}" ]]; then
+    pilotOptions+=" --pipInstallOptions=""${pip_install_options}"
```
Comment (chrisburr) [inline tests/CI/pilot_ci.sh:106]: This looks fishy with the redundant `""` in the middle, do you mean to do this:
```suggestion
    pilotOptions+=" --pipInstallOptions=${pip_install_options}"
```

Comment (chrisburr) [reply tests/CI/pilot_ci.sh:106]: Or do you mean this?
```suggestion
    pilotOptions+=" --pipInstallOptions='${pip_install_options}'"
```

Comment (fstagni) [reply tests/CI/pilot_ci.sh]: Fixed thanks.

**Review by chrisburr — APPROVED** [review]

**Pull request merged**